aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -240,22 +256,23 @@ void THNN_(SpatialConvolutionMM_updateGradInput)(
   int nOutputPlane = weight->size[0];
 
   int freeWeight = 0;
-  if (weight->nDimension == 4) {
+  if (weight && weight->nDimension == 4) {
```

Comment:
is this necessary? For the forward `weight` should never be NULL, right?

Reply:
Good point!